client/src/modules/stock/requisition/templates/available.cell.html

```diff
@@ -0,0 +1,21 @@
+<div class="ui-grid-cell-contents">
+  <span ng-if="grid.appScope.toggleAvailability && row.entity._initialised">
+    <span ng-if="row.entity.inventory.isAvailable">
+      <i 
+        translate-attr="{title : 'REQUISITION.INVENTORY_AVAILABLE'}" 
+        ng-if="row.entity.inventory.isEnough"
+        class="fa fa-thumbs-up text-success">
+      </i>
+      <i
+        translate-attr="{title : 'REQUISITION.INVENTORY_AVAILABLE_WITH_NOT_ENOUGH_QUANTITY'}"
+        ng-if="!row.entity.inventory.isEnough"
+        class="fa fa-warning text-warning">
+      </i>
+    </span>
+    <i 
+      translate-attr="{title : 'REQUISITION.INVENTORY_NOT_AVAILABLE'}" 
```

Comment:
Could you include the amount that is available in the target depot using a `uib-tooltip?`

![image](https://user-images.githubusercontent.com/896472/122216405-5efe7b00-ce7a-11eb-98f3-db2bf22f50ee.png)

That would make it much easier to validate that the information is correct from a user's standpoint.